src/components/characterList.jsx

```diff
@@ -26,8 +26,8 @@ const CharacterList = ({ currentScript }) => {
   // render list of characters and their data
   return (
     <div className='characterList'>
-      <h3>Characters in the play: </h3>
-      <ul>{charactersElements}</ul>
+      <h3>Characters in the play</h3>
+      <ul>{currentScript !== '' && charactersElements}</ul>
```

Comment:
This is so it will only render character elements if there is an active current script.

Reply:
Can we reset the `characters` state so it re-renders without the characters instead of skipping over it with this?